src/ocamlorg_frontend/pages/problems.eml

```diff
@@ -48,12 +53,22 @@ Learn_layout.render
         problems is available on
         <a href="https://github.com/ocaml/ocaml.org/tree/main/data/problems">GitHub</a>.
       </p>
+      <p>
+        Every exercise has a difficulty level, ranging from beginner to advanced. 
+        This difficulty level is indicated with the symbol "☡".
+        The absence of marks represents the beginner level, one mark the intermediate level and two marks the advanced level.
+      </p>
     </div>
     <% problems |> List.iter (fun (problem : Ood.Problem.t) -> %>
     <div x-data="{statement: true}" id="<%s" problem.number %>
       >
-      <div class="flex space-y-4 lg:space-y-0 lg:items-center flex-col lg:flex-row lg:justify-between">
-        <h4 class="font-bold text-body-600"><%s problem.title %></h4>
+      <div class="flex space-y-4 lg:space-y-0 flex-col lg:flex-row lg:justify-between">
+        <span aria-hidden="true" class="danger invisible lg:visible"><%s danger problem.difficulty %></span>
```

Comment:
```suggestion
        <span aria-hidden="true" class="danger invisible lg:visible" title="<%s Ood.Problem.difficulty_to_string problem.difficulty ^ " difficulty" %>"><%s danger problem.difficulty %></span>
```

Maybe we add a title too, so that people who don't read (like me) can still discover by hovering.